py/setup.cfg

```diff
@@ -0,0 +1,20 @@
+[metadata]
+name = serialkeel
+version = 0.1.0
+author = Torstein Grindvik
+author_email = torsteingrindvik@nordicsemi.no
+description = Serial Keel client for Python
+long_description = Testing various things
+long_description_content_type = text/markdown
+url = https://github.com/torsteingrindvik/serial-keel
+classifiers =
+    Programming Language :: Python :: 3
+    Operating System :: OS Independent
+[options]
+packages = find:
+python_requires = >=3.6
+include_package_data = True
+install_requires =
+	websockets
+	pytest-asyncio-cooperative
+
```

Comment:
Do we want this zephyr.hex in the repo?

Reply:
I see it's deleted and that's ok, so resolving this conversation.